openedx/core/djangoapps/user_api/accounts/tests/test_retirement_views.py

```diff
@@ -241,21 +243,29 @@ def test_user_can_deactivate_secondary_email(self):
         # Assert that there is no longer a secondary/recovery email for test user
         assert len(AccountRecovery.objects.filter(user_id=self.test_user.id)) == 0
 
-    def test_password_mismatch(self):
+    @mock.patch.object(NotJwtRestrictedApplication, 'has_permission')
+    @mock.patch.object(JwtAuthentication, 'authenticate')
+    def test_password_mismatch(self, jwt_authenticate, jwt_has_permissions):
```

Comment:
Is this the desired behavior here?
Same question for changes below.

Reply:
@jawad-khan yes. These test cases are written to test code in `user_api` djangoapp. JWT auth mocking is added so the code in here is properly covered/tested.